src/yang/ietf-access-control-list.yang

```diff
@@ -466,60 +466,71 @@ module ietf-access-control-list {
               "The rules in this set determine what fields will be
                matched upon before any action is taken on them.
                The rules are selected based on the feature set
-               defined by the server and the acl-type defined.";
-
-            container eth {
-              when "derived-from(../../../../acl-type, " +
-                   "'acl:eth-acl-type')";
-              if-feature match-on-eth;
-              uses packet-fields:acl-eth-header-fields;
+               defined by the server and the acl-type defined.
+               If the packet does not match any of the fields
+               defined, it is considered match for 'any'.";
+
+            choice l2 {
+              container eth {
+                when "derived-from(../../../../type, " +
+                     "'acl:eth-acl-type')";
+                if-feature match-on-eth;
+                uses packet-fields:acl-eth-header-fields;
+                description
+                  "Rule set that matches ethernet headers.";
+              }
               description
-                "Rule set that supports ethernet headers.";
+                "Choice of layer 2 headers, currently only
```

Comment:
You will never be able to remove "currently only Ethernet" in the description. If an augmentation later comes in and adds something else, this description is plain wrong.

I'd suggest changing to: `Match layer 2 fields, for example Ethernet header fields`